### ISSUE TITLE
cli: chatwoot integration does not respect inbox id

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2569]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.32.10](https://github.com/open-wa/wa-automate-nodejs/compare/4.32.9...4.32.10) (2022-03-16)

- 🐛 CLI: chatwoot integration inbox fix #2569 [`#2569`](https://github.com/open-wa/wa-automate-nodejs/issues/2569)
- Release 4.32.10 [`80d435f`](https://github.com/open-wa/wa-automate-nodejs/commit/80d435f164c033d28a4815845881ca8e503c8d7e)
- updated types-only package [`c43dfba`](https://github.com/open-wa/wa-automate-nodejs/commit/c43dfba6a848eb3582b508d82849d42ce0ce8890)

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.32.11](https://github.com/open-wa/wa-automate-nodejs/compare/4.32.10...4.32.11) (2022-03-16)

- Release 4.32.11 [`6f9e2c6`](https://github.com/open-wa/wa-automate-nodejs/commit/6f9e2c656ff18ed944e8718a320eccff652ce35f)
- updated types-only package [`1e75280`](https://github.com/open-wa/wa-automate-nodejs/commit/1e75280b2cd42f67776972b5529f5f8d835ca292)
